zstacklib/zstacklib/utils/lvm.py

```
@@ -2501,6 +2501,32 @@ def get_running_vm_root_volume_on_pv(vgUuid, pvUuids, checkIo=True):
 
+@bash.in_bash
+def get_running_vm_root_volume_on_vg(vgUuid):
+    # type: (str) -> list[VmStruct]
+    vms = []
+    for file_name in linux.listdir(LIVE_LIBVIRT_XML_DIR):
+        xs = file_name.split(".")
+        if len(xs) != 2 or xs[1] != "xml":
+            continue
+
+        xml = linux.read_file(os.path.join(LIVE_LIBVIRT_XML_DIR, file_name))
+        if not '/dev/' + vgUuid in xml:
```

Ruff (0.12.2)

Test for membership should be `not in`

Convert to `not in`

(E713)